v2/component/launcher.go

```diff
@@ -280,20 +309,26 @@ func (l *Launcher) prepareOutputs(ctx context.Context) error {
 // RunComponent runs the current KFP component using the specified command and
 // arguments.
 func (l *Launcher) RunComponent(ctx context.Context, cmd string, args ...string) error {
```

Comment:
nit: move public methods to the top of the file?

Reply:
Done.